packages/expo-cli/src/commands/export.ts

```diff
@@ -1,17 +1,15 @@
 import { ProjectTarget, getDefaultTarget } from '@expo/config';
-import { Project, ProjectSettings, UrlUtils } from '@expo/xdl';
-import axios from 'axios';
+import { Project, UrlUtils } from '@expo/xdl';
 import { Command } from 'commander';
 import crypto from 'crypto';
 import fs from 'fs-extra';
 import path from 'path';
-import targz from 'targz';
 import validator from 'validator';
 
 import CommandError from '../CommandError';
-import { installExitHooks } from '../exit';
 import log from '../log';
 import prompt, { Question } from '../prompt';
+import { downloadAndDecompressAsync } from './upload/submission-service/utils/files';
```

Comment:
It's really weird we're importing this function from Submission Service.
I'd prefer to create some utils directory/file in the root directory and move the function there.